bundle/tests/presets/databricks.yml

```diff
@@ -0,0 +1,22 @@
+bundle:
+  name: presets
+
+presets:
+  tags:
+    prod: true
+    team: finance
+  pipelines_development: true
+
+targets:
+  dev:
+    presets:
+      prefix: "myprefix"
+      pipelines_development: true
+      trigger_pause_status: PAUSED
+      jobs_max_concurrent_runs: 10
+      tags:
+        dev: true
+        prod: false
+  prod:
+    presets:
+      pipelines_development: false
```

Comment:
I prefer to keep the tests closer to the code and just make sure the model is processed correctly in these tests. But we have a bit of a special case here since we have this sociable unit test for presets above 🤷